MediaBrowser.Controller/Session/ISessionManager.cs

```diff
@@ -111,7 +111,8 @@ public interface ISessionManager
         /// Reports the session ended.
         /// </summary>
         /// <param name="sessionId">The session identifier.</param>
-        void ReportSessionEnded(string sessionId);
+        /// <returns>Task.</returns>
+        ValueTask ReportSessionEnded(string sessionId);
```

Comment:
Why ValueTask?

Reply:
Because the AsyncDisposable may or may not return a task, for this usecase ValueTask was made.